### PR TITLE
Handle `-- Mark --` logs in `rsyslog-relp-echo-server`'s output when checking if the extension is disabled

### PR DESCRIPTION
**Pull request by plkokanov**
**How to categorize this PR?**

/kind flake

**What this PR does / why we need it**:
This PR makes it so that `-- Mark --` messages, which are output by the `rsyslog-relp-echo-server` to indicate rsyslog is working properly, are taken into account when checking if no logs are sent from the nodes to the echo server: [ref](https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/1edcd373b1e7614a68d0fe4d3dbb1952c31b4d58/test/common/verifier.go#L169-L174).

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: test/common/verifier.go [hunk 1]
```diff
@@ -170,7 +170,7 @@ func (v *Verifier) verifyThatLogsAreNotForwardedToEchoServer(ctx context.Context
 	EventuallyWithOffset(2, func(g Gomega) {
 		logLines, err := v.getLogs(ctx, timeBeforeLogGeneration)
 		g.Expect(err).NotTo(HaveOccurred())
-		g.Expect(logLines).To(BeEmpty())
+		g.Expect(logLines).To(Or(BeEmpty(), ConsistOf(ContainSubstring("-- Mark --"))))
```
Comment (ialidzhikov) [inline test/common/verifier.go:173]: Optionally, we can optionally add a comment describing the `-- Mark --` logs as I didn't know what is the reason behind it:

```suggestion
		// -- Mark -- is a period log emitted by rsyslogd denoting that there are no incoming logs.
		g.Expect(logLines).To(Or(BeEmpty(), ConsistOf(ContainSubstring("-- Mark --"))))
```

Comment (plkokanov) [reply test/common/verifier.go:173]: The more correct description would be that it is a "heartbeat" log message which lets you know that it is working. I'll add it for clarity.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**